### PR TITLE
Prism fixes

### PR DESCRIPTION
**Pull request by christopherjbaker**
fixes canjs/canjs#3447

### CODE REVIEW

File: prettify.js [hunk 1]
```diff
@@ -1,33 +1,80 @@
 require("./prettify.less");
 
+require("./prism-config");
 var Prism = require("prismjs");
 require("prismjs/plugins/line-numbers/prism-line-numbers.js");
```
Comment (justinbmeyer) [inline prettify.js]: generally w/ steal, you don't include the `.js`

File: prettify.js [hunk 2]
```diff
@@ -1,33 +1,80 @@
...
- *
- * Also requires [bit-docs-prettify/prettify.less].
+ * Get node for provided line number
+ * Copied from prism-line-numbers.js and modified to support nested spans
+ * @param {Element} element pre element
+ * @param {Number} number line number
+ * @return {Element|undefined}
  */
-module.exports = function(){
+Prism.plugins.lineNumbers.getLine = function (element, number) {
```
Comment (justinbmeyer) [inline prettify.js]: can you add some docs explaining this?

File: prettify.js (hunk 1 above)
Comment (christopherjbaker) [reply prettify.js]: fixed

File: prettify.js (hunk 2 above)
Comment (christopherjbaker) [reply prettify.js]: there was a bit, but I added more to the docblock above. also, this has been moved to `bit-docs-html-highlight-line`

**Pull request merged**